### PR TITLE
WIP: Qcloud

### PR DESCRIPTION
**Pull request by richard1122**

### CODE REVIEW

**Comment by richard1122** [comment]
1. 以后 provider 可以放在海外服务器上了，最好可以和 wukong 放一起
1. 发请求时用了 qcloud 的云函数，用消息队列把请求内容推过去，那边发完请求之后再塞回消息队列，provider 会订阅消息队列收返回。
1. redis 直接在 docker 里面开个算了

**Comment by Senorsen** [comment]
Okay but a Dockerfile is needed

File: src/providers/NeteaseCloudMusic.ts [hunk 1]
```diff
@@ -15,8 +14,8 @@ class NeteaseCloudMusicProvider extends BaseMusicProvider {
         return 'netease-cloud-music'
     }
 
-    static apiPrefix = serverConfig['netease-cloud-music-api-prefix']
-    static binCdn = serverConfig['netease-cloud-music-bin-cdn']
+    static apiPrefix = 'http://music.163.com'
```
Comment (Senorsen) [inline src/providers/NeteaseCloudMusic.ts]: Currently https://music.163.com is a better choice

**Comment by Leeleo3x** [comment]
Hi @richard1122, Could you please fix the breaking tests? Thx.

**Comment by richard1122** [comment]
@Leeleo3x no

**Comment by Leeleo3x** [comment]
Chat with @richard1122 offline and we decided to bu guan this failure.

**Pull request merged**